src/test/java/edu/ucsb/cs156/frontiers/controllers/RepositoryControllerTests.java

```diff
@@ -113,6 +102,25 @@ public void job_actually_fires() throws Exception {
         assertEquals(expectedJson, actualJson);
     }
 
+    @Test
+    @WithInstructorCoursePermissions
```

Comment:
One of the reasons that I asked for the javadoc is that while I can sort of "guess" what this means, I'm not 100% confident than I do know what it means, or that I can explain to someone how it works.